src/python/benchUtil.py

```diff
@@ -1554,6 +1554,7 @@ def getClassPath(checkout):
     raise RuntimeError('can\'t find core JAR file in %s' % ('%s/lucene/core/build/libs' % path))
 
   cp.append(core_jar_file)
+  cp.append('%s/lucene/core/build/classes/java/test' % path)
```

Comment:
I suppose this is the most controversial one. knnPerfTest needs `KnnGraphTester` which is in the `test` directory. I can move this into just the `knnPerfTest` if necessary.

Reply:
I think it's OK? Maybe add a TODO linking to an issue about moving KnnGraphTester out of test?